src/NuGet.Clients/NuGet.PackageManagement.UI/Converters/SubstractionConverter.cs

```diff
@@ -27,7 +27,7 @@ public object Convert(object[] values, Type targetType, object parameter, Cultur
             {
                 return 0;
             }
-            return System.Convert.ToDouble(values[0]) - System.Convert.ToDouble(values[1]) - System.Convert.ToDouble(values[2]);
+            return System.Convert.ToDouble(values[0], CultureInfo.CurrentCulture) - System.Convert.ToDouble(values[1], CultureInfo.CurrentCulture) - System.Convert.ToDouble(values[2], CultureInfo.CurrentCulture);
```

Comment:
Input values seems to be doubles: https://cs.github.com/NuGet/NuGet.Client/blob/e0db7dcd14807fd1f8874df18e9a7c7d9cd871f6/src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml#L85

An explicit cast is sufficient. Don't forget to validate array length in line 26.

```suggestion
            if (values[0] is double v0 && values[1] is double v1 && values[2] is double v2)
            {
              return v0 - v1 - v2;
            }
            
            return 0;
```